scripts/database/upgrades/upgrade_v4.10.1_to_v4.10.2.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE externaltool ADD COLUMN contenttype text;

```

Comment:
Two things:

Should this have a `NOT NULL` in it to be consistent with `@Column(nullable = false)`?

Are we sure the next version is 4.10.2? This may be a question for @djbrooke .